OrbitGl/TimerTrack.cpp

```diff
@@ -107,14 +107,12 @@ void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
 
         bool is_visible_width = normalized_length * canvas->GetWidth() > 1;
         bool is_selected = &text_box == GOrbitApp->selected_text_box();
-        bool need_highlight = GOrbitApp->selected_text_box() &&
-                              timer_info.thread_id() != SamplingProfiler::kAllThreadsFakeTid &&
-                              timer_info.thread_id() == GOrbitApp->selected_thread_id() &&
-                              timer_info.function_address() ==
-                                  GOrbitApp->selected_text_box()->GetTimerInfo().function_address();
-        need_highlight =
-            need_highlight ||
-            (!GOrbitApp->selected_text_box() && GOrbitApp->highlighted_function() > 0 &&
+        bool need_highlight =
+            (GOrbitApp->selected_text_box() && /*if picking from the capture window*/
```

Comment:
Maybe a better way to implement this is by making variables.
bool picked_from_capture_windows = ....
bool picked_from_live_func_panel = ....
bool need_highlight = picked_from_capture_windows || picked_from_live_func_panel 

And you don't need comments. What do you think?

Reply:
Good suggest! Will do that : )